CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,84 @@
+# gittuf Contributor ladder
+
+gittuf's contributor ladder is loosely inspired by the OpenSSF [Scorecard
+Contributor Ladder]. This document details how community members may advance to
+more-involved roles within the gittuf project.
+
+- [Roles](#roles)
+    - [Community members](#community-members)
+    - [Maintainers](#maintainers)
+- [Inactive members](#inactive-members)
+
+## Roles
+
+### Community members
+
+Community members engage with gittuf, contributing their time and energy in
+discussions or just generally helping out.
+
+#### Pre-requisites
+
+- Must follow the [OpenSSF Code of Conduct]
+- Must follow the [Contribution Guide]
+
+#### Responsibilities
+
+- Keep it up!
+
+### Maintainers
+
+Maintainers are responsible for the project's overall health. They are the only
+ones who can approve and merge code contributions.
+
+**Defined by:** "Maintain" permissions in the project
+
+#### Pre-requisites
+
+- Reviewed at least 10 substantial PRs to the codebase
+- Reviewed or got at least 30 PRs merged to the codebase
+
+#### Responsibilities
+
+- Demonstrate sound technical judgment
+- Maintain project quality control via code reviews
+    - Focus on holistic acceptance of contribution
+- Be responsive to review requests
+- Mentor contributors
+- Approve and merge code contributions as appropriate
+- Participate in gittuf community meetings, if possible
+- Facilitating gittuf-specific community meetings, if possible and comfortable
+
+#### Privileges
+
+- Maintainer status may be a precondition to accepting especially large code
```

Comment:
I'd take this out as a  whole, I'm not sure we want to grant this privilege (even though I'm aware I've been guilty of this on occasion).